00_ES6/00_start/src/api/routes.js

```diff
@@ -0,0 +1,7 @@
+// TODO move to some setting/env place
+
+const baseUrl = 'http://localhost:3000';
+const baseApi = baseUrl + '/api/';
```

Comment:
Why not backtics?

Reply:
I was not sure, in this case we have no variable to interpolate (only baseUrl), but you are right probably it will look more readable, I will update it.

Reply:
Updated, thanks for the feedback